links.html

```diff
@@ -1,41 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-    
-   
-    </head>
-    
-    <body>
-    <header><nav>
-    
-    <ul>
-    <li><a href="index.html">Home</a></li>
-    <li><a href=" links.html">Link</a></li>
-    <li><a href="glossary.html">Glossary</a></li> 
-    </ul>
-   </nav>
-   </header>
+<head>
+    
+    <title>YAOTIAN TONG's CSD120 Website</title>
+    
+</head>
+    
+<body>
+    <header>
+        <nav>    
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Link</a></li>
+                <li><a href="glossary.html">Glossary</a></li> 
+            </ul>
+        </nav>
+    </header>
    
     <main>
     
     <h1>Useful Links</h1>
     
-   	<dd><li><a href="https://validator.w3.org/nu/">HTML Validator</a></li></dd>
+   	<ul><li><a href="https://validator.w3.org/nu/" target="_blank">HTML Validator</a></li></ul>
```

Comment:
Use `<ul>` to mark up an unordered list .
Use the `target` attribute to make links open in a new tab